src/sessionmanagerplugin/session/session.go

```diff
@@ -190,7 +190,7 @@ func ValidateInputAndStartSession(args []string, out io.Writer) {
 
 //Execute create data channel and start the session
 func (s *Session) Execute(log log.T) (err error) {
-	fmt.Fprintf(os.Stdout, "\nStarting session with SessionId: %s\n", s.SessionId)
+	fmt.Fprintf(os.Stderr, "\nStarting session with SessionId: %s\n", s.SessionId)
```

Comment:
Many just notification messages like these. e.g. Start/stop session, or call out KMS enabled. Not sure changing the behavior to write to Stderr is a better solution.

Reply:
Maybe a quiet mode to not print those messages

Reply:
It's better behaviour because it doesn't mix the output with normal output. Say, you call a remote script to generate CSV. Right now you end up with log messages mixed with CSV output. With this change, the log messages go to stderr and aren't mixed with stdout.

Reply:
Yep, it causes issues like this: https://github.com/zagaran/eb-ssm/pull/4/files#diff-e2c321f9e0a8996da965d12461b0e7d1216893f21adf5fde6cf7366fe288fe55R113

Reply:
Thanks all, will test it and support the change in next release.

Reply:
Hi, just a quick update, we cannot merge this change as plan, since many customer are trying to extract info from the Stdout. e.g. plugin version. This change will break their current use experience. As @tapajos mentioned, we may need a quiet mode, or reduce some logs, change part of them to Stderr. Team will work on resolve this issue in a graceful way later. Thanks for understanding.

Reply:
LOL, I am in awe of what some people do... yes, please introduce a quiet mode. What is the ETA?

Reply:
@Yangtao-Hua would #94 provide what you need to avoid breaking current user experience?